stdlib/2and3/wsgiref/types.pyi

```diff
@@ -15,27 +15,34 @@
 # you need to use 'WSGIApplication' and not simply WSGIApplication when type
 # hinting your code.  Otherwise Python will raise NameErrors.
 
-import sys
-from typing import Callable, Dict, Iterable, List, Optional, Tuple, Type, Union, Any
+from typing import Callable, Dict, Iterable, List, Optional, Tuple, Type, Union, Any, Text, Protocol
 from types import TracebackType
 
 _exc_info = Tuple[Optional[Type[BaseException]],
                   Optional[BaseException],
                   Optional[TracebackType]]
-if sys.version_info < (3,):
-    _Text = Union[unicode, str]
-    _BText = _Text
-else:
-    _Text = str
-    _BText = Union[bytes, str]
-WSGIEnvironment = Dict[_Text, Any]
+_BText = Union[bytes, str]
```

Comment:
This is just `str` in Python 2. Is that right?

Reply:
It isn't. This should actually only be bytes. Fixing.